src/components/offer-card/OfferCard.styles.js

```diff
@@ -0,0 +1,96 @@
+import { commonShadow } from '~/styles/app-theme/custom-shadows'
+
+import { blueGrey, green } from '@mui/material/colors'
+
+export const styles = {
+  container: {
+    display: 'flex',
+    padding: '30px 20px',
+    boxShadow: commonShadow
+  },
+  leftContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    mr: '20px'
+  },
+  avatar: {
+    height: '80px',
+    width: '80px',
+    mb: '16px'
+  },
+  ratingContainer: {
+    display: 'flex',
+    borderRadius: '8px',
+    padding: '6px',
+    backgroundColor: 'basic.grey'
+  },
+  rating: {
+    mr: '5px'
+  },
+  bio: {
+    display: '-webkit-box',
+    WebkitLineClamp: 2,
+    lineClamp: 2,
+    WebkitBoxOrient: 'vertical',
+    boxOrient: 'vertical',
+    overflow: 'hidden',
+    mb: '10px'
+  },
+  chipsContainer: {
+    display: 'flex',
+    mb: '10px'
+  },
+  subjectChip: {
+    mr: '4px',
+    backgroundColor: green[300]
+  },
+  description: {
```

Comment:
allmost the same as bio, move all repeated fields to variable and reuse

Reply:
Done.